### PR TITLE
ran isort autopep and docformat. No functionality change

### PR DESCRIPTION
**Pull request by sajal-g**

### CODE REVIEW

File: src/main/py/hl7_listener/main.py [hunk 1]
```diff
@@ -1,22 +1,25 @@
-"""
-This HL7 MLLP Listener/Receiver Service will do the following:
-1) Connect to the configured HL7 MLLP host and then listen for incoming HL7 messages.
-2) Received messages will be sent to the configured NATS JetStream server Subject. If the message
-   send to the NATS server fails, the process of listening for incomming HL7 messages will halt.
+"""This HL7 MLLP Listener/Receiver Service will do the following: 1) Connect to the
```
Comment (Palisand) [inline src/main/py/hl7_listener/main.py:1]: I liked it how you had it before. Maybe it has to do with no period or blank line after `following`?

**Review by Palisand — APPROVED** [review]

File: src/main/py/hl7_listener/main.py (hunk 1 above)
Comment (sajal-g) [reply src/main/py/hl7_listener/main.py]: added a blank line and it looks better now

File: src/main/py/hl7_listener/main.py [hunk 2]
```diff
@@ -1,22 +1,26 @@
-"""
-This HL7 MLLP Listener/Receiver Service will do the following:
+"""This HL7 MLLP Listener/Receiver Service will do the following:
+
 1) Connect to the configured HL7 MLLP host and then listen for incoming HL7 messages.
-2) Received messages will be sent to the configured NATS JetStream server Subject. If the message
-   send to the NATS server fails, the process of listening for incomming HL7 messages will halt.
+2) Received messageswill be sent to the configured NATS JetStream server Subject. If the message send to the
```
Comment (Palisand) [inline src/main/py/hl7_listener/main.py:4]:
```suggestion
2) Received messages will be sent to the configured NATS JetStream server Subject. If the message sent to the
```

**Review by Palisand — DISMISSED** [review]

File: src/main/py/hl7_listener/main.py (hunk 2 above)
Comment (sajal-g) [reply src/main/py/hl7_listener/main.py]: done

**Review by Palisand — APPROVED** [review]

**Pull request merged**